packages/calling-stateful-client/src/Converter.ts

```diff
@@ -148,7 +141,7 @@ export function convertFromSDKToDeclarativeVideoStreamRendererView(
 /**
  * @private
  */
-export const isACSCall = (call: Call | /* @conditional-compile-remove(teams-call) */ TeamsCall): call is Call => {
+export const isACSCall = (call: CallCommon): call is Call => {
```

Comment:
Consider extracting the `isAcsCall*` type functions to their own file. That way `Converter.ts` can be completely agnostic of Teams/ACS differences. These functions aren't helping the conversion as such, and keeping them here increases the cognitive load reading understanding this file.